### ISSUE TITLE
Prefers column/row is broken

### ISSUE THREAD

**Issue by vyeevani** [google-deepmind/treescope#15]
Prefers column/row is currently broken in the ArrayAutoVisualizer
```
    row_axes = []
    column_axes = []
    for info in array_axis_info:
      if isinstance(info, NamedPositionalAxisInfo | NamedPositionlessAxisInfo):
        if info.axis_name in self.prefers_column:
          column_axes.append(info.axis_name)
        elif info.axis_name in self.prefers_row:
          row_axes.append(info.axis_name)
```
should be
```
    row_axes = []
    column_axes = []
    for info in array_axis_info:
      if isinstance(info, NamedPositionalAxisInfo | NamedPositionlessAxisInfo):
        if info.axis_name in self.prefers_column:
          column_axes.append(info)
        elif info.axis_name in self.prefers_row:
          row_axes.append(info)
```

**Comment by vyeevani** [comment]
Opened PR: https://github.com/google-deepmind/treescope/pull/17

**Comment by danieldjohnson** [comment]
Good catch, thanks!

**Comment by danieldjohnson** [comment]
I've fixed the original issue in release 0.1.3. (I've left #17 open since it seems like it has some other changes in it as well.)
